addons/skin.estuary/xml/DialogVideoManager.xml

```diff
@@ -5,83 +5,81 @@
 		<control type="group">
 			<centertop>50%</centertop>
 			<centerleft>50%</centerleft>
-			<width>1320</width>
-			<height>710</height>
+			<width>1640</width>
+			<height>780</height>
 			<include>Animation_DialogPopupVisible</include>
 			<include content="DialogBackgroundCommons">
-				<param name="width" value="1320" />
-				<param name="height" value="710" />
+				<param name="width" value="1640" />
+				<param name="height" value="780" />
 				<param name="header_id" value="2" />
 			</include>
 			<control type="image">
-				<top>200</top>
-				<left>40</left>
-				<width>260</width>
-				<height>390</height>
-				<align>center</align>
-				<aligny>center</aligny>
-				<aspectratio>keep</aspectratio>
-				<texture fallback="DefaultVideo.png" background="true">$VAR[VideoVersionPosterVar]</texture>
+				<left>20</left>
+				<width>400</width>
+				<height>630</height>
+				<top>100</top>
+				<texture>$VAR[VideoListPosterVar]</texture>
+				<aspectratio aligny="top">keep</aspectratio>
+			</control>
+			<control type="label">
+				<description>Path label</description>
+				<left>20</left>
+				<bottom>10</bottom>
+				<width>1300</width>
+				<height>30</height>
+				<font>font12</font>
+				<align>right</align>
+				<aligny>bottom</aligny>
+				<textcolor>grey</textcolor>
+				<label>$INFO[Container(50).ListItem.FileNameAndPath]</label>
```

Comment:
Probably needs a `<scroll>true</scroll>` for this label.

Reply:
Didn't want a scroll as the right alignment means last part of path is always seen.